### PR TITLE
ci: added linter

### PR DESCRIPTION
**Pull request by vojtechsimetka**
Added CircleCI and linter rules according to what we agreed in our [naming discussion](https://hackmd.io/9rbfNqHwTpSw3ZZKJDXekQ). The linting rules follow the [tasegir linter rules](https://github.com/AuHau/tasegir/blob/master/config/eslintrc.js) because we can not overwrite them from the config and hence can't use the tasegir directly.

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -1,54 +1,108 @@
...
+    }
+  },
+  'extends': ['standard', 'plugin:@typescript-eslint/recommended', 'airbnb'],
+  'env': {
+    'browser': true,
+    'commonjs': true,
+    'es6': true,
+    'jest': true,
+    'node': true,
+    'mocha': true
```
Comment (jurajpiar) [inline .eslintrc:16]: don't think we're using mocha at all in this project.

**Comment by julianmrodri** [comment]
This is great! Bunch of fixes that need to be done but really cool feature

**Comment by vojtechsimetka** [comment]
Yes this test is flaky, PR #30 is removing the test as that part is fully rewritten.

**Comment by jurajpiar** [comment]
As the CI check says: change the last commit type.

**Pull request merged**